SlackHistoryViewer.Helpers/Database.cs

```diff
@@ -5,19 +5,104 @@
 using Newtonsoft.Json;
 using SlackHistoryViewer.Database;
 using SlackHistoryViewer.Database.Models;
-using SlackHistoryViewer.Helpers;
 using SlackHistoryViewer.Configuration;
 using SlackHistoryViewer.Slack.Models;
 
-namespace SlackHistoryViewer.RestoreFromLocal
+namespace SlackHistoryViewer.Helpers
```

Comment:
the `Database` should not be in the helpers. each console can have a different database class with the same name and different methods.

modify this accordingly.